pdp/static/js/__test__/pdp_controls.test.js

```diff
@@ -0,0 +1,71 @@
+require('./globals-helpers').importGlobals([
+    { module: 'js/jquery-1.10.2.js', name: ['$', 'jQuery'] },
+    { module: 'js/lodash.custom.js', name: ['_', 'lodash'] },
+    { module: 'js/condExport.js', name: 'condExport' },
+    { module: 'js/classes.js', name: 'classes' },
+    { module: 'js/calendars.js', name: 'calendars' },
+    { module: 'js/pdp_controls.js', spread: true },
+], '../..');
+
+
+describe('RasterDownloadLink', function() {
+    document.body.innerHTML =
+        '<div>' +
+            '<a id="download-timeseries" href="#"/>' +
+        '</div>';
+
+    var ncwmsLayer = {
+        events: {
+            register: function() {}
+        }
+    };
+
+    describe('mock app setup', function() {
+        var dlLink;
+        beforeEach(function() {
+            dlLink = new RasterDownloadLink(
+                $('#download-timeseries'),
+                ncwmsLayer,
+                undefined,
+                'nc',
+                'tasmax',
+                't0:t1',
+                'y0:y1',
+                'x0:x1'
+            );
+        });
+
+        afterEach(function() {
+            dlLink = null;
+        });
+
+        it('has the supplied initial time range', function() {
+            expect(dlLink.trange).toBe('t0:t1');
+        });
+
+        it('generates the expected download URL', function() {
+           expect(dlLink.getUrl()).toBe('.nc?tasmax[t0:t1][y0:y1][x0:x1]&')
```

Comment:
Just checking, but this test generates a download URL with an undefined filename?